randovania/games/samus_returns/json_data/header.txt

```diff
@@ -125,23 +143,29 @@ Dock Weaknesses
       Open:
           Trivial
       Lock type: FRONT_BLAST_BACK_IF_MATCHING
-          Lay Power Bomb or Shoot Wave Beam
+          Any of the following:
+              Shoot Wave Beam
+              Knowledge (Beginner) and Lay Power Bomb
 
 
   * Spazer Beam Door
       Extra - type: spazer_beam
       Open:
           Trivial
       Lock type: FRONT_BLAST_BACK_IF_MATCHING
-          Lay Power Bomb or Shoot Spazer Beam
+          Any of the following:
+              Shoot Spazer Beam
+              Knowledge (Beginner) and Lay Power Bomb
 
 
   * Plasma Beam Door
       Extra - type: plasma_beam
       Open:
           Trivial
       Lock type: FRONT_BLAST_BACK_IF_MATCHING
-          Plasma Beam or Lay Power Bomb
+          Any of the following:
+              Plasma Beam
```

Comment:
Out of curiosity why is this not a template?

Reply:
Plasma is the last beam in the sequence so it doesn't need to list other beams

Reply:
Wouldn't this be better tho for future proofing for whenever split beams become a thing?

Reply:
How is split beams relevant? Plasma doors can't be opened by Wave or Spazer.

Reply:
Right, whoops